qiskit/tools/_compiler.py

```diff
@@ -182,6 +182,7 @@ def execute(circuits, backend,
         qobj_id (int): identifier for the generated qobj
         hpc (dict): HPC simulator parameters
         skip_transpiler (bool): skip most of the compile steps and produce qobj directly
+        kwargs: extra arguments used by AER for runing configurable backends
```

Comment:
Maybe add:

"Refer to the backend documentation for details on these arguments"